tests/test_commands.py

```diff
@@ -316,7 +316,7 @@ def _make_read_only(path: Path):
         self.assertEqual(actual_permissions, expected_permissions)
 
     def test_startproject_permissions_unchanged_in_destination(self):
-        """Check that pre-existing folders and files in the destination folder
+        """Check that preexisting folders and files in the destination folder
```

Comment:
This one seems arguable, but no strong opinion either way.

Same applies to “nonexistent” later on.